graphViz/main.c

```diff
@@ -0,0 +1,64 @@
+#pragma warning(disable : 4996)
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define GRAPH_SIZE 5
+#define EXPRESSION_SIZE 100
+
+typedef enum
+{
+    success,
+    fileError
+} ErrorCode;
+
+int main()
+{
+    char* dotExePath = getenv("GraphVizDot");
+    if (dotExePath == NULL)
+    {
+        dotExePath = "C:\\Program Files\\Graphviz\\bin\\dot.exe"; // default path to dot.exe
+    }
+
+    const char matrix[GRAPH_SIZE][GRAPH_SIZE] = { {0, 1, 0, 0, 0},
+                                                  {0, 0, 1, 0, 0},
+                                                  {0, 0, 0, 1, 0},
+                                                  {0, 0, 0, 0, 1},
+                                                  {1, 0, 0, 0, 0} };
+    char dotFileName[] = "graph.dot";
+
+    FILE* file = NULL;
+    fopen_s(&file, dotFileName, "w");
+    if (file == NULL)
+    {
+        return fileError;
+    }
+
+    fprintf(file, "digraph Graph {\n");
```

Comment:
```suggestion
    fprintf(file, "digraph {\n");
```
Если так сделать, то работает :thinking: